### PR TITLE
Fall events update

### PR DESCRIPTION
**Pull request by viviehn**
Still need to update fall workshops page

### CODE REVIEW

File: compete/rc-fall.html [hunk 1]
```diff
@@ -9,101 +9,118 @@
...
-	</tr>
-	<tr>
-	    <th scope="row">5:00 PM</th>
-	    <td colspan="2">Close</td>
-	</tr>
-</table>
-</div>
-</div>
+
+  <!--<div class="panel-heading">Schedule</div>-->
```
Comment (kev-chien) [inline compete/rc-fall.html]: These comments look painful (and painful to remove). Or did you use an IDE?

File: programs/past-programs.md [hunk 2]
```diff
@@ -3,6 +3,15 @@ layout: markdown
 hero-image: url(../assets/images/stock-images/prep.jpg)
 hero-message: Past Programs
 ---
+# Fall 2016 Workshops
+
+## Building Bridges with Pioneers in Engineering
+
+The first in our series of Fall Workshops, Building Bridges with Pioneers in Engineering took place on campus on Saturday, September 24th from 9:45 AM to 3:00 PM in O'Brien Breezeway. Students from all backgrounds and experiences had tool training, learned about prototyping, design, and participated in designing and constructing their own bridge.
```
Comment (kev-chien) [inline programs/past-programs.md]: I would add photos to these programs. And... either eliminate the wall of text, or put the photos at the side of the text. Ideally the formatting would be... nicer haha.

Comment (kev-chien) [reply programs/past-programs.md]: Thoughts:

- Could have blog, and simple "cards" that lead to these blog posts. And blog posts that feed to Facebook. Oh wait, we do have that huh.

- More random thoughts I'll just type out here:
- - PiE Snapchat presence?
- - PiE Facebook presence could be vastly improved?
- - Event links to Facebook event on front page?
- - Encourage other Pie Staff to comment "issues" on the website?

Sorry for the brain dump here haha

File: programs/past-programs.md [hunk 3]
```diff
@@ -3,6 +3,15 @@ layout: markdown
...
+
+## Creating Code with PiE
+PiE hosted its second workshop at UC Berkeley on Saturday, September 29th from 10 AM to 3:00 PM in Soda Hall. While our workshop is intended for students with no programming experience, students from all backgrounds and experiences are appreciated and very welcome! At this workshop, students learned about basic programming logic, variables, conditionals and loops, functions, and get a taste of recursion -- all in Scratch! All computers were provided.
```
Comment (kev-chien) [inline programs/past-programs.md]: This was for... high school or UCB students? Or both came? If so, maybe write "students from all backgrounds and experiences were welcomed, including high school students and UCB students". Also, consistent past tense is preferred (imo).

File: compete/rc-fall.html (hunk 1 above)
Comment (viviehn) [reply compete/rc-fall.html]: Ah it's a vim plugin i think lol

File: programs/past-programs.md (hunk 3 above)
Comment (viviehn) [reply programs/past-programs.md]: The workshops were only for high school students

Comment (viviehn) [reply programs/past-programs.md]: Will change all to past tense!

**Comment by kev-chien** [comment]
Wait, I don't have write access haha.

**Comment by viviehn** [comment]
OOPS
now you do

**Pull request merged**